.github/workflows/release.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      # - uses: pipe-cd/actions-gh-release@v2.3.6
+      # - uses: pipe-cd/actions-gh-release@v2.4.0
```

Comment:
this is not being used anyway, so how about removing this file?

Reply:
Exactly, let's remove this.